lms/djangoapps/courseware/access.py

```diff
@@ -878,12 +878,18 @@ def get_user_role(user, course_key):
     """
     role = get_masquerade_role(user, course_key)
 
-
     # Check if user is a coach on this ccx.
-    ccx_coach_role = False # Added by labster
+    ccx_coach_role = False  # Added by labster
     if isinstance(course_key, CCXLocator):
         try:
-            ccx_coach_role = has_ccx_coach_role(user, course_key)
+            ccx = CustomCourseForEdX.objects.get(pk=course_key.ccx)
```

Comment:
So what is the fix?
The only differnce I see, is that `ccx_id` is wrapped by `str`. Is it a fix?